app/lib/rocketchat.js

```diff
@@ -1032,7 +1032,27 @@ const RocketChat = {
 		// RC 0.42.0
 		return this.methodCallWrapper('getUsersOfRoom', rid, allUsers, { skip, limit });
 	},
+	getMembers(rid, roomType, type, debouncedText, start = 0, end = 10) {
+		const params = {
+			roomId: rid,
+			offset: start,
+			count: end,
+			...(type !== 'all' && { 'status[]': type }),
+			...(debouncedText && { filter: debouncedText })
+		};
+		// RC 3.16.0
+		if (roomType === 'd') {
+			return this.sdk.get('im.members', params);
+		}
 
+		if (roomType === 'p') {
+			return this.sdk.get('groups.members', params);
+		}
+
+		if (roomType === 'c') {
+			return this.sdk.get('channels.members', params);
+		}
```

Comment:
You should use `roomTypeToApiType` for this, like https://github.com/RocketChat/Rocket.Chat.ReactNative/blob/develop/app/lib/rocketchat.js#L1056